### DISCUSSION TITLE
Where are the moderators?

### DISCUSSION THREAD

**Discussion by sovetski** [community/community#55652] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

There are just a few examples of discussions without any context:

https://github.com/orgs/community/discussions/55160
https://github.com/orgs/community/discussions/55560
https://github.com/orgs/community/discussions/55072
https://github.com/orgs/community/discussions/54985
https://github.com/orgs/community/discussions/54769

It creates pollution, we need moderators who check the subjects before validating them

**Comment by queenofcorgis** [comment, marked as answer]
Hi there, 

Thanks for flagging our way, we appreciate you helping keep the community clutter-free. I want to caveat that our moderator team does not validate these posts. In the future, when you see a post that you think needs moderator attention, feel free to [report it](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-discussion) for our review. Appreciate again you surfacing these posts @sovetski 💟 .
